### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mzr**
.

### CODE REVIEW

**Comment by dingorth** [comment]
@melouber What do you think?

**Comment by dingorth** [comment]
```Haskell
instance Monad Maybe where
   return = Just
   Just x >>= f = f x
```

**Comment by dingorth** [comment]
Look at this new commit @melouber !

File: task1/task1.c [hunk 1]
```diff
@@ -1,9 +1,9 @@
-#include <stdio.h>
-#include <stdlib.h>
-#include <stdint.h>
-
-int clz(long);
+#include <stdio.h>	// including headers
```
Comment (melouber) [inline task1/task1.c]: Very nice comment!

**Comment by melouber** [comment]
@dingorth

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # university-assembly-tasks
```
Comment (melouber) [inline README.md]: Wow!

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # university-assembly-tasks
-x86-64 assembly task list for university course Computer Systems Architecture
+## x86-64 assembly task list for university course Computer Systems Architecture
 * task1 - clz bit function
 * task2 - lcm & gcd
 * task3 - insert sort
```
Comment (melouber) [inline README.md]: Great!

File: task1/task1.c [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+#include <stdlib.h>	// including headers
+#include <stdint.h>	// including headers
 
+int clz(long);
```
Comment (melouber) [inline task1/task1.c]: Not so nice.

**Review by jpszczolowski — APPROVED** [review]
.

**Comment by melouber** [comment]
I changed my decision. Your changes are terrible. @dingorth

File: task1/task1.c (hunk 1 above)
Comment (dingorth) [reply task1/task1.c]: Thanks! I was doing my best here.

File: README.md (hunk 2 above)
Comment (dingorth) [reply README.md]: Indeed. Wow.

**Pull request closed without merging**